boards/riscv32/arty_litex_vexriscv/doc/arty_litex_vexriscv.rst

```diff
@@ -0,0 +1,33 @@
+.. _arty-litex-vexriscv:
+
+LiteX VexRiscv
+##############
+
+Overview
+********
+
+LiteX is a Migen-based System on Chip, supporting various softcore CPUs,
+including VexRiscv. The LiteX SoC with VexRiscv CPU can be deployed on e.g.
+Digilent ARTY board. More information can be found on:
+`LiteX's website <https://github.com/enjoy-digital/litex>`_ and
+`VexRiscv's website <https://github.com/SpinalHDL/VexRiscv>`_.
+
+Programming and debugging
+*************************
+
+Building
+========
+
+Applications for the ``arty_litex_vexriscv`` board configuration can be built as usual
+(see :ref:`build_an_application`).
+In order to build the application for ``arty_litex_vexriscv``, set the ``BOARD`` variable
+to ``arty_litex_vexriscv``.
+
+Booting
+=======
+
+You can boot from serial port using `flterm: <https://github.com/timvideos/flterm>`_, e.g.:
```

Comment:
First I've heard of this tool (flterm).  Does it do something that other serial port tools we typically mention don't do?

Reply:
LiteX (SoC) has a built-in ROM bootloader ("bios") that supports uploading binaries but a special tool (e.g. flterm) is needed.